source/plot/SvgPlot.ooc

```diff
@@ -104,7 +105,7 @@ SvgPlot: class {
 		}
 	}
 	fillColorList: func {
-		this colorList = VectorList<String> new()
+		this colorList = VectorList<String> new(10, false)
```

Comment:
I think `false` means that they are ntot freed. They are string literals as described in the issue description. It is another weird thing in this language. I tried to reproduce this in a simple test case and 
```ooc
test := "string"
test2 := "string"
```
actually allocates two separate objects, while here they all points to static literals. I don't know what triggers each mode.